include/ddc/kernels/splines/spline_builder_batched.hpp

```diff
@@ -127,9 +181,17 @@ class SplineBuilderBatched
         return spline_tr_domain_type(bsplines_domain(), batch_domain());
     }
 
-    int offset() const noexcept
+    /**
+     * @brief Get the interpolation matrix.
+     *
+     * Get the interpolation matrix. This can be useful for debugging (as it allows
+     * one to print the matrix) or for more complex quadrature schemes.
+     *
+     * @return A reference to the interpolation matrix.
+     */
+    const Matrix& get_interpolation_matrix() const noexcept
     {
-        return spline_builder.offset();
+        return *matrix;
     }
```

Comment:
Not really in the scope of the MR but I needed it for [MR ](https://gitlab.maisondelasimulation.fr/gysela-developpers/gyselalibxx/-/merge_requests/359)